mx.nodeprof/mx_nodeprof.py

```diff
@@ -22,66 +22,66 @@ def prepareJalangiCmdLine(args):
 
 class OutputCapture:
     def __init__(self, outFile):
-        self.outFile = outFile;
+        self.outFile = outFile
```

Comment:
Do we have actual changes needed in this file except for the format change?
It would be great if we can keep the change minimal and only touch necessary files for clarity.
In future, we may integrate with some auto-formatting tool like `black` to get rid of such Python formatting issues.